docs/upload-image.md

```diff
@@ -68,6 +68,29 @@ If Rancher is deployed on an RKE2 cluster, perform the following steps:
 
 3. Delete the stuck image, and then restart the upload process.
 
+#### Upload Huge Size Image Pending in Rancher Multi-Cluster Management
+
+If you upload an image with huge size (usually more than 10GB) from Multi-Cluster Management, the image may stay in status *Uploading*.
+
+This is related to the *proxy-request-buffering* in ingress configuration, which is also specific to the cluster that is hosting Rancher.
```

Comment:
Hi @jillian-maroket,
I changed `If you upload a very large image (over 10 GB) from the Multi-Cluster Management screen on the Rancher UI, the Uploading status may be displayed for an extended period.` as `If you upload a very large image (over 10 GB) from the Multi-Cluster Management screen on the Rancher UI, the progress may be stuck in the Uploading status.` to avoid misleading, please take a look, thanks.

Reply:
@WebberHuang1118 I understand that my suggestion does not describe the issue well. Unfortunately, the new wording "the progress may be stuck in the Uploading status" is not grammatically correct. How about this:

*If you upload a very large image (over 10 GB) from the **Multi-Cluster Management** screen on the Rancher UI, the operation may take longer than usual and the image status (*Uploading*) may not change.*

Reply:
@jillian-maroket 
Updated per suggestion, thanks.